### ISSUE TITLE
Require that "*/" go on its own line for multiline blocks

### ISSUE THREAD

**Issue by danvk** [gajus/eslint-plugin-jsdoc#738]
## Motivation

I'd like to require that the trailing `*/` in multiline comments go on its own line:

Bad:
```js
/**
 * This is a multline
 * jsdoc comment with a bad ending */
```

Good:
```js
/**
 * This is a multline
 * jsdoc comment with a good ending
 */
```

## Current behavior

The `multiline-blocks` rule distinguishes single and multiline block forms, but does not enforce that the ending `*/` be on its own line for multiline blocks.

## Desired behavior

I'd like to enforce that the `*/` go on its own line for multiline comments.

So this would be an error:

```js
/**
 * This is a multline
 * jsdoc comment with a bad ending */
```

It's important to allow this for one line comments, hwoever:

```js
/** This is a single line jsdoc comment with a fine ending. */
```

## Alternatives considered

The `no-multi-asterisks` rule feels somewhat related but also does not enforce this. See https://github.com/gajus/eslint-plugin-jsdoc/issues/199#issuecomment-841838326

**Comment by brettz9** [comment]
Sounds good...I'd like to add this as a `noFinalLineText` option on `multiline-blocks` (as a complement to the existing `noZeroLineText` option).

**Comment by brettz9** [comment]
By the way, although `no-multi-asterisks` currently changes:

```js
      /**
       * Desc.
       *
       * */
```

to the misaligned:

```js
      /**
       * Desc.
       *
        */
```

...I think this is ok because the whitespace was indeed there and we have the `check-alignment` rule to handle the alignment.

For the proposed `noFinalLineText`, I similarly don't think the rule should be checking for such cases of a delimiter and whitespace preceding the `*/`--only for non-whitespace text since the other rules handle this.

**Comment by danvk** [comment]
Confirmed that this is working exactly as I'd hoped on my repo. Thanks so much @brettz9!
